docs/source/tutorials/camera_tuning.rst

```diff
@@ -0,0 +1,27 @@
+Camera tuning
+=============
+
+Our library supports setting camera IQ (Image Quality) tuning blob, which would be used for all cameras.
+By default, cameras will use a *general* tuning blob, which works great in most cases - so changing the camera
+tuning blob is not needed for most cases.
+
+.. code-block:: python
+
+    import depthai as dai
+
+    pipeline = dai.Pipeline()
+    pipeline.setCameraTuningBlobPath('/path/to/tuning.bin')
+
+Available tuning blobs
+######################
+
+To tune your own camera sensors, one would need Intel's software, for which a license is needed
+- so the majority of people will only be able to use pre-tuned blobs.
+
+**Currently available tuning blobs:**
+
+- Color tuning for low-light environments `here <https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/misc/tuning_color_low_light.bin>`__. Comparison below.
```

Comment:
Other details to add here: This also allows auto-exposure to go up to 100ms (otherwise limited with default tuning to 33ms), but needs the FPS to be reduced, for example `camRgb.setFps(10)`.
Known limitation: flicker can be seen with auto-exposure over 33ms, it is caused by auto-focus working in continuous mode. A workaround is to change from CONTINUOUS_VIDEO (default) to AUTO (focusing only once at init, and on further focus trigger commands):
`camRgb.initialControl.setAutoFocusMode(dai.CameraControl.AutoFocusMode.AUTO)`